packages/miniflare/src/plugins/kv/assets.ts

```diff
@@ -97,3 +98,122 @@ export function getAssetsServices(options: AssetsOptions): Service[] {
 	};
 	return [storageService, namespaceService];
 }
+
+const MAX_ASSET_COUNT = 20_000;
```

Comment:
could we please extract all the assets manifest related logic in a separate file 🙏 ?

Reply:
Hmm actually I would prefer to keep this all in this file to prevent it getting too fragmented. But I've added some nice big comment headers to segment out the file in a hopefully sensible way.